src/app/components/ms-item/ms-item.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="section" [class.has-nested]="nested" [class.has-nested2]="nested2">   
+    <div class="row no-gutters" *ngIf="data?.locus">
```

Comment:
If you want to use bootstrap, use it correctly. Check the grid layout rules at https://getbootstrap.com/docs/4.1/layout/overview/ 

N.B. In our case it's better to use "container-fluid". Check bootstrap guidelines